mysite/settings.py

```diff
@@ -0,0 +1,122 @@
+"""
+Django settings for mysite project.
+
+Generated by 'django-admin startproject' using Django 1.10.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.10/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.10/ref/settings/
+"""
```

Comment:
you don't really need this now. Leave only first string `Django settings for mysite project.`